### PR TITLE
Fix documentation for SSH

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixed and reorder steps in documentation

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,9 @@ An ssh-bastion pod to make access to openshift clusters easy
...
-    export this variable SSH_KEY_PATH to change its location. (e.g `export SSH_KEY_PATH=~/.ssh/libra.pem`).
 1. Run: `curl https://raw.githubusercontent.com/eparis/ssh-bastion/master/deploy/deploy.sh | bash`
-1. ssh as core to/through the bastion.
+1. SSH as core user to/through the bastion.
+    https://raw.githubusercontent.com/eparis/ssh-bastion/master/ssh.sh provides a simple script to automate sshing through to bastion to nodes in the cluster.
+    > If your SSH key used during deploy of OCP is different from default one you are using, you can
+    > export this variable `SSH_KEY_PATH` to change its location. (e.g `export SSH_KEY_PATH=~/.ssh/libra.pem`).
 1. The bastion address can be found by running `oc get service -n openshift-ssh-bastion ssh-bastion -o jsonpath='{.status.loadBalancer.ingress[0].hostname}'`
-
-https://raw.githubusercontent.com/eparis/ssh-bastion/master/ssh.sh provides a simple script to automate sshing through to bastion to nodes in the cluster.
```
Comment (eparis) [inline README.md:14]: Why would I want to remove this comment?

Comment (petr-balogh) [reply README.md:14]: @eparis I didn't removed it, just moved to line 9 as you can see.

Comment (eparis) [reply README.md:14]: ok, yeah, i'm dumb. if you rebase i'll merge. sorry.

Comment (petr-balogh) [reply README.md:14]: @eparis rebased, thanks

**Pull request merged**